docs/api.yml

```diff
@@ -60,6 +60,39 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/Book'
+
```

Comment:
/books/{status}
status: path parameters
Create a path parameter that can have the values "borrowed" and "available"